examples/hybrid_jobs/0_Creating_your_first_Hybrid_Job/Creating_your_first_Hybrid_Job.ipynb

```diff
@@ -38,20 +38,16 @@
    ]
   },
   {
-   "cell_type": "code",
-   "execution_count": 1,
-   "metadata": {},
-   "outputs": [
-    {
-     "name": "stdout",
-     "output_type": "stream",
-     "text": [
-      "This block is a copy of the algorithm script.\n"
-     ]
+   "cell_type": "markdown",
+   "metadata": {
+    "pycharm": {
+     "name": "#%% md\n"
     }
-   ],
+   },
    "source": [
-    "%%script echo \"This block is a copy of the algorithm script.\"\n",
+    "#### This block is a copy of the algorithm script.\n",
+    "\n",
+    "```python\n",
```

Comment:
So, it's an interesting thing. This file has a `%%script echo \"This block is a copy of the algorithm script.\"` line in one of the cells, which fails when run in python 3.8. I don't know why, exactly, since it's a script call from the notebook, but it was failing both the tests and when I switched to 3.8 and ran it manually. My belief was that whoever wrote this didn't want that cell executed and wrote the script call to prevent the execution. If so, I think that's not a good way to do it. 

The change converts that whole cell to a markdown cell and quotes the python code so it's not executable at all which, I think, was the original intention.